lambda.dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM hayd/deno-lambda:1.6.1
+
+COPY . .
+RUN for i in $(find . -name "*.ts" -not -name "*_test.ts"); do \
+    deno cache --unstable $i; \
+  done
```

Comment:
Can you add a deno run deps.ts here to cache the mongo plugin during builds?

Reply:
Isn't already being taken care of since it's imported by the different entrypoints?

Reply:
No. The plugin shared object is downloaded to disk on first run of the program.

Reply:
If you do this in a few steps it is much better for local dev:

```Dockerfile
COPY deps.ts .  # also copy and cache test_deps.ts here?
RUN deno run -A --unstable deps.ts

COPY . .
RUN deno cache *.ts  # not sure if this syntax works!
```

Since deps.ts changes less frequently the deps are downloaded/compiled and plugin object downloaded in a cached STEP, and only rerun if deps.ts changes, which means you don't have to redo that slow step every time.